src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1342,7 +1342,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         if (!_midiAudio)
         {
-            _midiAudio = std::make_unique<MidiAudio>();
+            const auto windowHandle = reinterpret_cast<HWND>(_owningHwnd);

```

Comment:
note for @zadjii-msft: this will need to be changed when tearout changes the owning HWND